thorium/response.py

```diff
@@ -71,11 +71,15 @@ def __init__(self, *args, **kwargs):
 
     def get_response_data(self):
         data = []
-        if self.resources:
+        if getattr(self, 'resources', False):
+            data = []
             self._sort()
             self._paginate()
             for res in self.resources:
                 data.append(OrderedDict(res.sorted_items()))
+        if getattr(self, 'resource', False):
+            data = None
+            data = OrderedDict(self.resource.sorted_items())
```

Comment:
I need to here because custom actions are done though post requests. So sometimes it will be a collection or sometimes its just one item. So the result will be put in either resources or resource. I need to check them. Then I need  to set the default to be [] in case it doesn't have either. Some older test cases were failing without it.